src/runtime/cgo_noos.go

```diff
@@ -6,7 +6,9 @@ package runtime
 
 import "unsafe"
 
-const iscgo = false
+var iscgo bool
+
+func init() { iscgo = false }
```

Comment:
This function does nothing :)

Reply:
The cgo is a delicate topic in context of small architectures. Making iscgo variable instead of a constant may increase the runtime significantly which will affect almost all MCU targets.